techniques/failures/F109.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+ 
+ <head><title>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</title>
+   
+ <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></head><body><h1>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</h1><section class="meta"><p class="id">ID: F109</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
+      <p>All technologies that require authentication.</p>
+   </section><section id="description"><h2>Description</h2>
+  <p>Requiring users to authenticate by entering a password in a different format from which it was originally created is a failure to meet Success Criteria 3.3.8 and 3.3.9 (unless alternative authenticaton methods are available). Requiring users to enter individual characters of a single password across multiple fields prevents users pasting credentials from a password manager or local copy of the password. This means users cannot avoid transcription, resulting in a <a href="../../understanding/22/accessible-authentication.html#dfn-cognitive-function-test">cognitive function test</a>. This applies irrespective of whether users are required to enter all characters in the password, or a subset thereof.</p>
+  
+  
+   </section><section id="examples"><h2>Examples</h2>
+  <p>These examples would prevent a user from filling a password in the same format in which the password was origianlly created:</p>
+     <ul>
+				<li>A fieldset that prompts a user to "Enter the 2nd, 6th and last characters of your password", with separate input fields for each character</li>
+				<li>A fieldset that prompts a user to enter all digits of a 6-digit passcode, across 6 individual fields.</li>
```

Comment:
A common pattern I see today with SMS-based 2FA providers is a 6-digit one-time code spread across 6 individual fields, but with custom paste behavior that allows a user to paste the code into the first field despite the use of multiple fields.

Do we think this would meet the requirement? If so, we should consider an "unless" clause here and in the second sentence of line 8

Reply:
sure, i'd add it to that second list item at the end `(unless the user is able to paste the entire 6-digit passcode, and the individual fields are filled in automatically)` or something

Reply:
Good clarification. I've rephrased line 8, and added this clarification to the second list item